puppet/modules/scaleway/templates/manage_snapshots.erb

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/ruby
```

Comment:
This file should be deployed in `init.pp`. It can probably also become a static file rather than a template.

Reply:
Done